src/pages/AppLogin.vue

```diff
@@ -25,15 +25,19 @@ const email = ref('')
 const hasValidEmail = computed(() => isValidEmail(email.value))
 
 async function launchLogin(type: string) {
-  if (type === 'passwordless') {
-    if (!hasValidEmail.value) return
-    loaderStore.showLoader(`Sending login link to your email`)
-    await arcanaAuth.loginWithLink(email.value)
-  } else {
-    loaderStore.showLoader(`Signing with ${type}`)
-    await arcanaAuth.loginWithSocial(type)
+  try {
+    if (type === 'passwordless') {
+      if (!hasValidEmail.value) return
+      loaderStore.showLoader(`Sending login link to your email`)
+      await arcanaAuth.loginWithLink(email.value)
+    } else {
+      loaderStore.showLoader(`Signing with ${type}`)
+      await arcanaAuth.loginWithSocial(type)
+    }
+    await fetchAndStoreDetails()
+  } catch (e) {
+    loaderStore.hideLoader()
```

Comment:
Do we also need to hide the loader if there's no error thrown?

Reply:
We have a hideLoader inside `fetchAndStoreDetails`

Do we need to take it outside like example below?
```js
await fetchAndStoreDetails()
loaderStore.hideLoader()
```